defaults/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
-# defaults file for tinc
-
-service_name: "{{ tinc_service_name }}"
+tinc_netname: tinc-vpn
+tinc_control_plane_bind_ip: "{{ ansible_eth0.ipv4.address }}"
```

Comment:
isn't this risky, as not everyone has a nic named eth0? 
Instead I would use the following as default: `"{{ ansible_default_ipv4.address }}"`

Also, can this be done in a separate commit? I would like to see just the minimum required for molecule first :)